rootfs/api/tests/test_hooks.py

```diff
@@ -166,10 +166,10 @@ def test_build_hook_dockerfile(self):
         build = {'username': 'autotest', 'app': app_id}
         url = '/v2/hooks/builds'.format(**locals())
         SHA = 'ecdff91c57a0b9ab82e89634df87e293d259a3aa'
-        DOCKERFILE = """
-        FROM busybox
-        CMD /bin/true
-        """
+        DOCKERFILE = """\
+FROM busybox
+CMD /bin/true"""
```

Comment:
my guess would be the tab is missing from the old one, but moving `FROM busybox` to after """ should make the tests pass:

```
"""FROM busybox
CMD /bin/true
"""

looks like some odd formatting hack in the build hook, but that should stop the bleeding.
```